samples/net/sockets/echo_client/overlay-ot.conf

```diff
@@ -24,6 +24,7 @@ CONFIG_OPENTHREAD_L2_DEBUG=y
 CONFIG_OPENTHREAD_L2_LOG_LEVEL_INF=y
 
 CONFIG_OPENTHREAD_CHANNEL=26
+CONFIG_OPENTHREAD_MASTERKEY="00:11:22:33:44:55:66:77:88:99:aa:bb:cc:dd:ee:ff"
```

Comment:
Couldn't we set this string as a default for `OPENTHREAD_MASTERKEY` option?

Reply:
That would be against the intention of the OpenThread PR of not having a default Master Key: https://github.com/openthread/openthread/pull/5998

Reply:
Ok, understood. Please update other OT overlays that we have in the tree then:
https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/net/lwm2m_client/overlay-ot.conf
https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/subsys/mgmt/updatehub/overlay-ot.conf

Reply:
Thanks for reminding about the other samples. The FOTA one won't require the Master Key since it's using Commissioning.